src/utils/account/account.module.ts

```diff
@@ -1,11 +1,10 @@
 import { Module } from '@nestjs/common';
 import { AccountService } from './account.service';
-import { MsGraphModule } from '@utils/ms-graph/ms-graph.module';
 import { UserModule } from '@domain/user/user.module';
 import { AccountResolver } from './account.resolver';
 
 @Module({
-  imports: [MsGraphModule, UserModule],
+  imports: [UserModule],
```

Comment:
For later: ideally Account should not have a dependency on User service? Basically two pieces of logic that use the userservice need to be moved outside. Actually the updates re register / apply might also make the code that uses userservice redundant...

Reply:
Agreed - that needs a discussion. Out-of-scope for now thou - otherwise this PR will be 5 epics into 1.